content/components/recipes/form/arguments/PreferenceRollback.tsx

```diff
@@ -19,12 +19,30 @@ export default function PreferenceRollback() {
       });
   }, [environmentKey]);
 
-  const rolloutOptions = rollouts.map((rollout) => ({
-    label: rollout.latest_revision.arguments.slug,
+  const rolloutData: Array<{
+    label: string;
+    value: string;
+  }> = rollouts.map((rollout) => ({
+    label: `${rollout.id}: ${rollout.latest_revision.name} `,
     value: rollout.latest_revision.arguments.slug,
   }));
 
+  const rolloutMenu = (label, item): ReactElement => {
+    return (
+      <div>
+        {label}
+        <small style={{ opacity: 0.5, fontSize: "95%" }}>{item.value}</small>
```

Comment:
I think this should be `<small className="text-subtle">` instead?

Reply:
you're right, i'm such a goof 🙈